data/assets/scene/digitaluniverse/grids.asset

```diff
@@ -338,19 +334,20 @@ local plane100ly = {
     Rotation = {
       Type = "StaticRotation",
       Rotation = eclipticRotationMatrix
+    },
+    Scale = {
```

Comment:
Same here, I think defining the size to be `2 * 10 * lightYear` might be easier to understand

Reply:
Same for the ones below;  just to reduce the level of complexity by removing the need for specifying the `StaticScale` at all